### DISCUSSION TITLE
White screen on start at Ubuntu 22.04 other than English system locale

### DISCUSSION THREAD

**Discussion by ilkeroktem** [rustdesk/rustdesk#3359] [Q&A]
Hello everyone,

When we try to run Rustdesk on Ubuntu 22.04 Desktop with Turkish UI, it returns blank white screen. I could only find a link here: https://www.reddit.com/r/rustdesk/comments/wb0jab/rustdesk_showing_a_white_screen_workaround_linux/

And he/she is right! When I change system locale to English, it works normally.

I tried to run it from terminal with Turkish UI, it shows the result as below:

"
ERROR:TIS: Error: bad syntax : Expecting a primary expression
ERROR:TIS: line:var svg_checkmark = ;
ERROR:TIS: here:____________________^
ERROR:TIS: file:(index.html(132))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
INFO:TIS: warning:aspect 'Progress' not found
INFO:TIS: warning:aspect 'Progress' not found
INFO:TIS: warning:aspect 'Progress' not found
INFO:TIS: warning:aspect 'Progress' not found
ERROR:DOM: Creation of behavior edit was rejected by <input.search_id}> element. Wrong DOM model.
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
ERROR:DOM: Creation of behavior edit was rejected by <input.outline-focus> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input.outline-focus> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input.outline-focus> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input.outline-focus> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input.input}> element. Wrong DOM model.
ERROR:DOM: Creation of behavior password was rejected by <input.outline-focus> element. Wrong DOM model.
ERROR:DOM: Creation of behavior password was rejected by <input> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input#remote_id.outline-focus> element. Wrong DOM model.
ERROR:DOM: Creation of behavior edit was rejected by <input.outline-focus> element. Wrong DOM model.
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
WARNING:CSS: unrecognized property name syntax at (index.html(1))
"

I cannot change to English UI because computer's user just does not want.

Any suggestions would be appreciated.

**Comment by rustdesk** [comment]
try out nightly build

**Reply by ilkeroktem** [reply]
Thank you, nightly build starts normally. But I can't connect it from my 1.1.9 version. Do I have to have nightly build 1.2 too, and can 1.2 connect other 1.1.9 ones?
